src/Controller/Back/PartnerController.php

```diff
@@ -538,7 +540,11 @@ public function checkMail(
             }
 
             $validator = new EmailValidator();
-            $emailValid = $validator->isValid($request->get('email'), new RFCValidation());
+            $multipleValidations = new MultipleValidationWithAnd([
```

Comment:
Il ne faudrait pas utiliser `EmailFormatValidator` pour avoir le même fonctionnement partout ?

Reply:
oui ça peut être pas mal, mais dans ce cas, il faut que je renforce EmailFormatValidator car il laisse passer `machin@orange.f`

Reply:
Oui, je ne l'ai pas généralisé partout, mais peut-être qu'il faudrait le faire dans ce ticket ? :thinking: 
Ce qui veut dire appeler la route checkmail depuis le nouveau formulaire aussi

Reply:
Bien qu'il n'existe pas d'extension officielle avec une seule lettre, il est techniquement possible de créer une adresse email avec une extension d'une seule lettre, comme le montre cette discussion sur Stack Overflow : 

[Is it possible to have one single character top-level domain name?](https://stackoverflow.com/questions/7411255/is-it-possible-to-have-one-single-character-top-level-domain-name).

Cela explique le fonctionnement de la RFCValidation.

**Je recommande d'éviter les appels DNS afin de maintenir le contrôle sur la validation et de permettre à la plateforme de fonctionner en local sans connexion Internet.**

Les erreurs restent marginal, si l'agent ne reçoit pas le mail,il peut nous contacter ou contacter son RT. 
On peut aussi anticiper en vérifiant sur Sentry.

**Vérifier que les extensions ont plus de deux lettres est suffisant pour le moment.**

SI on doit renforcer le contrôle, je suis plus pour une validation 2FA pour vérifier le mail de l'agent mais ça dépasse le cadre de ce ticket

Coté usager il me semble qu'on ait pas eu de plainte sur la réception des emails dernièrement

Reply:
Ok j'ai fait ces changements